src/cdk-experimental/menu/menu-item.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, Output, Input, OnDestroy} from '@angular/core';
+import {Subject} from 'rxjs';
+import {CdkMenuPanel} from './menu-panel';
+
+/**
+ * Directive which provides behaviour for an element which when clicked either:
+ *  - executes the user defined click handler
+ *  - toggles it's checkbox state
+ *  - toggles it's radio button state (in relation to siblings)
+ *  - opens up an attached submenu
```

Comment:
does this one happen on click or hover? I would expect hover

Reply:
It depends on where it exists. I'll make it more clear